week1.js3/index.js

```diff
@@ -0,0 +1,52 @@
+
+async function fetchData() {
+    try {
+        const response = await fetch("https://raw.githubusercontent.com/najmeh25/najmeh25.github.io/refs/heads/main/index.json");
+        allData = await response.json();  
+        console.log(allData);  
+        displayData(allData);  
```

Comment:
fetchData should only have one responsibility and that is fetching the data. Consider moving `displayData` to be called after `fetchData`